bodhi/client/__init__.py

```diff
@@ -298,6 +298,8 @@ def overrides():
 @overrides.command('query')
 @click.option('--user', default=None,
               help='Updates submitted by a specific user')
+@click.option('--mine', 'user', flag_value=os.getenv('USERNAME'),
+              help='Updates submitted by current user')
```

Comment:
If we add ". user can be set with USERNAME environment variable" to the help text, it'll be consistent with how the ```--url``` flag is documented.